docs/recipes/amp-alertmanager-terraform.md

```diff
@@ -0,0 +1,141 @@
+# Terraform as Infrastructure as a Code to deploy Amazon Managed Service for Prometheus and configure Alert manager
+
+In this recipe, we will demonstrate how you can use [Terraform](https://www.terraform.io/) to provision [Amazon Managed Service for Prometheus](https://aws.amazon.com/prometheus/) and configure rules management and alert manager to send notification to a [SNS](https://docs.aws.amazon.com/sns/) topic if a certain condition is met.
+
+
+!!! note
+    This guide will take approximately 30 minutes to complete.
+
+## Prerequisites
+You will need the following to complete the steps in this blog post:
+* [Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) 
+* [AWS CLI version 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+* [Terraform CLI](https://www.terraform.io/downloads)
+* [AWS Distro for Opentelemetry](https://aws.amazon.com/blogs/mt/amazon-managed-service-for-prometheus-is-now-generally-available/)
+* [eksctl](https://eksctl.io/)
+* [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html)
+* [jq](https://stedolan.github.io/jq/download/%20)
+* [helm](https://www.eksworkshop.com/beginner/060_helm/helm_intro/install/index.html)
+* [SNS topic]
+
+In the recipe, we will use a sample application in order to demonstrate the metric scraping using ADOT and remote write the metrics to the workspace. Fork and clone the sample app from the repository at aws-otel-community (https://github.com/aws-observability/aws-otel-community). Execute the below commands to deploy application inside the EKS cluster. Make sure the terminal has access to the EKS cluster, follow the link (https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-connection/) to connect to EKS cluster
```

Comment:
First time mention ADOT, spell it out (or intro the abbreviation in pre-req section).

Reply:
You can drop the last sentence "Make sure the terminal …"